kolibri/plugins/management/assets/src/vue/manage-class-page/class-create-modal.vue

```diff
@@ -85,6 +86,9 @@
       },
     },
     vuex: {
+      getters: {
+        facility: state => state.facility,
+      },
```

Comment:
No need to add this `facility` dependency into the Vue file. It can instead be directly referenced inside the `createClass` action:

```javascript
  function createClass(store, stateClassData) {
    const classData = {
      name: stateClassData.name,
      parent: store.state.facilityId,
    };
    ...
```